scripts/fledge

```diff
@@ -636,6 +637,116 @@ fledge_purge() {
     source "$FLEDGE_ROOT/scripts/storage" purge
 }
 
+# Perform a healthcheck on the Fledge instance
+fledge_healthcheck() {
+
+	result=`fledge_status "silent"`
+	if [[ $result != "0" ]]; then
+		fledge_log "info" "Fledge appears not to be running, healthcheck should be performed on a running system . Start Fledge first." "all" "pretty"
+		exit 0
+	fi
+
+	echo Fledge Healthcheck
+	echo ==================
+	echo
+
+	auth="optional"
+	result=`curl -k -s ${REST_API_URL}/fledge/health/logging`
+
+	if [[ "${result}" == "401"* ]]; then
+		token=`fledge_authenticate`
+		if [[ "${token}" =~ "failed" ]]; then
+			fledge_log "info" "Failed authentication when attempting to stop fledge." "all" "pretty"
+			echo "Authentication failed."
+			exit 0
+		fi
+		result=`curl -k -s -H "authorization: $token" ${REST_API_URL}/fledge/health/logging`
+		auth="required"
+	fi
+
+	a=`echo $result | jq '.levels[] | {name,level}' | jq 'select (.level == "info") .name'`
+	if [[ "$a" != "" ]]; then
+		echo The following services have a logging level set to info, $a
+		echo This is probably too high and may result in large log files
+		echo
+	fi
+
+	a=`echo $result |jq '.levels[] | {name,level}' | jq 'select (.level == "debug") .name'`
+	if [[ "$a" != "" ]]; then
+		echo The following services have a logging level set to debug $a
+		echo This is probably too high and may result in large log files
+		echo
+	fi
+
+	a=`echo $result |jq 'select (.disk.usage > 80) .disk.usage'`
+	if [[ "$a" != "" ]]; then
+		echo The disk space in the logging directory is low, $a%
+		echo
+	fi
+
+	if [[ "${auth}" == "required" ]]; then
+		result=`curl -k -s -H "authorization: $token" ${REST_API_URL}/fledge/health/storage`
+	else
+		result=`curl -s ${REST_API_URL}/fledge/health/storage`
+	fi
+	a=`echo $result |jq 'select (.disk.usage > 80) .disk.usage'`
+	if [[ "$a" != "" ]]; then
+		echo The disk space in the storage directory is low, $a%
+		echo
+	fi
+
+	service rsyslog status >/dev/null
```

Comment:
Assuming that rsyslog is available for all Fledge supported platforms

Reply:
Without it we get no log output. This was an issue at BRP, hence the inclusion